tests/py_test.py

```diff
@@ -754,48 +752,62 @@ def make_monster_from_generated_code():
     ''' Use generated code to build the example Monster. '''
 
     b = flatbuffers.Builder(0)
-    string = b.CreateString("MyMonster")
```

Comment:
@rw As the building order is too complicated for the end users, so I remove the the old API, the new API would translate it to equivalent code automatically. As the result, the old API is no longer available for the end users. If we need more tests, we can just add more tests for the raw builder API.